commit b9bd56a756d42cdc7fc50ff45fe7ab7264421768 (refs/changes/79/379/3)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2016-09-29T01:24:50+00:00 (3 years ago)
    
    MANATEE-312 manatee should not set snapdir to visible
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Dave Pacheco <dap@joyent.com>

diff --git a/tools/mkdevsitters b/tools/mkdevsitters
index a6dc936..92b23aa 100755
--- a/tools/mkdevsitters
+++ b/tools/mkdevsitters
@@ -33,7 +33,7 @@ function mksitter
 
 	echo -n "peer $which: creating ZFS dataset ... "
 	if ! zfs create -o mountpoint=$mountpoint -o canmount=noauto \
-	    -o snapdir=visible $dataset; then
+	    $dataset; then
 		fail "failed to create dataset \"$dataset\""
 	fi
 
